mhw_analysis/systems/src/build.c

```diff
@@ -77,6 +78,21 @@ void first_pass(char *cube, int *mask, int *shape, int *parent, int *category) {
     int maxnlabels = 100000000;
     int this_label;
 
+    // Ocean boundaries
+    int indian_lon0 = 580; // Approximately 145E
+    int indian_lat0 = 360; // Approximately 0deg
+    int indian_lon1 = 400; // Approximately 100E
+    int indian_lat1a = 360; // Approximately 0deg
+    int indian_lat1b = 484; // Approximately 31degN
+    int indian_lon2 = 80; // Approximately 20E
+    int indian_lat2 = 360; // Approximately 0deg
+
+    int pacific_lon0 = 1160; // Approximately 70W
+    int pacific_lat0 = 360; // Approximately 0deg
+    int pacific_lon1a = 400; // Approximately 70W
+    int pacific_lon1b = 1040; // Approximately 70W
+    int pacific_lat1 = 624; // Approximately 0deg
+
```

Comment:
it might be helpful to put these into a struct which would contain all the relevant information for lats/lons/other constants to make this code cleaner

Reply:
Also a good suggestion.  Same response as above.